dame_flame/generate_new_active_sets.py

```diff
@@ -40,7 +40,7 @@ def algo3GenerateNewActiveSets(newly_dropped, prev_processed):
             rho.add(i)
 
     # Step 5: s_e is the support of covariate e in delta_k, for covars in rho
-    # TODO: combine this with above loop, calculating rho, for time efficiency.
```

Comment:
can you write that this is a "suggested change" rather than just dropping the todo?

Reply:
Catching todos is really important though, thanks! Would look super bad to reviewers and users if it looked unfinished. Thanks again!